src/System.Private.ServiceModel/tests/Common/src/BridgeClient.cs

```diff
@@ -50,8 +50,10 @@ private static void MakeConfigRequest()
             using (HttpClient httpClient = new HttpClient())
             {
                 httpClient.BaseAddress = new Uri(BridgeBaseAddress);
+                string resourceFolder = TestProperties.GetProperty(TestProperties.BridgeResourceFolder_PropertyName);
+                string contentPayload = "{ resourcesDirectory : \"" + resourceFolder + "\" }"; 
```

Comment:
To put squigglies in string.Format you just need to double them up: string.Format("{{ resources: \"{0}\" }}"